great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -980,6 +980,7 @@ def create_expectation_suite(
     def delete_expectation_suite(
         self,
         expectation_suite_name: Optional[str] = None,
+        ge_cloud_id: Optional[str] = None,
```

Comment:
This is already merged but I had a question about this:
1. It doesn't look like we do anything with `ge_cloud_id`. Is this to prep for future work?
2. We have `expectation_suite_name` as `Optional` but it looks like we immediately `raise` if the `has_key(key)` fails where key is the name. Should we remove the `Optional` annotation on this argument?